### PR TITLE
python3Packages.python-colout: init at 0.12.0

### PR DESCRIPTION
**Pull request by badele**
###### Description of changes

colout python module read lines of text stream on the standard input and output characters matching a given regular expression PATTERN in given COLOR and STYLE.

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/colout/default.nix
Comment (RaitoBezarius) [inline pkgs/development/python-modules/colout/default.nix:11]: Is there no way to build it from source?

Comment (badele) [reply pkgs/development/python-modules/colout/default.nix:11]: obviously no => https://pypi.org/project/colout/#files

Comment (RaitoBezarius) [reply pkgs/development/python-modules/colout/default.nix:11]: PyPI do not have sometimes the records for source, but GitHub seems to have it here: https://github.com/nojhan/colout/blob/master/setup.py

Comment (RaitoBezarius) [reply pkgs/development/python-modules/colout/default.nix:11]: (otherwise, we have to mark the provenance of this package as binary-only which would be a shame)

**Comment by RaitoBezarius** [comment]
Result of `nixpkgs-review pr 197946` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>3 packages built:</summary>
  <ul>
    <li>darwin.builder</li>
    <li>python310Packages.colout</li>
    <li>python39Packages.colout</li>
  </ul>
</details>

**Comment by RaitoBezarius** [comment]
Can the commit be named `python3Packages.python-colout` rather? Thanks!

File: pkgs/development/python-modules/colout/default.nix
Comment (RaitoBezarius) [inline pkgs/development/python-modules/colout/default.nix:33]: can we have a `pythonImportsCheck = [ "colout" ];` (I suppose this is how we import it)?

File: pkgs/development/python-modules/colout/default.nix
Comment (RaitoBezarius) [inline pkgs/development/python-modules/colout/default.nix:34]: Can you add a comment on why tests have to be disabled?

**Comment by RaitoBezarius** [comment]
Hi, we are close! Can you split the commit where you introduce your maintainer record and the package commit? Then, we can merge.

**Comment by RaitoBezarius** [comment]
Result of `nixpkgs-review pr 197946` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>4 packages built:</summary>
  <ul>
    <li>python310Packages.colout</li>
    <li>python310Packages.colout.dist</li>
    <li>python311Packages.colout</li>
    <li>python311Packages.colout.dist</li>
  </ul>
</details>

**Review by RaitoBezarius — APPROVED** [review]

**Comment by RaitoBezarius** [comment]
The maintainer commit is not formatted according to the CONTRIBUTING guide, please can you reformat it? We are good to go after this.

**Pull request merged**
